components/org.wso2.carbon.identity.user.account.association.stub/src/main/resources/UserAccountAssociationService.wsdl

```diff
@@ -6,6 +6,9 @@
             <xs:complexType name="UserAccountAssociationDTO">
                 <xs:sequence>
                     <xs:element minOccurs="0" name="domain" nillable="true" type="xs:string"/>
+                    <xs:element minOccurs="0" name="email" nillable="true" type="xs:string"/>
```

Comment:
Let's try to avoid API changes. Better to add new API without affecting to the existing APIs.

Reply:
As discussed offline, we will keep this as it is. Since the WSDL file is updated accordingly to match with the admin service, client stub can be generated for existing clients.